src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -36,18 +37,35 @@ export default class DiscussionArea {
   /**
    * Creates a `DiscussionArea` for a `lecture`.
    */
-  constructor(lecture) {
+  constructor(lecture, eventController) {
     this.#lecture = lecture;
+    this.#eventController = eventController;
     this.#manager = new DiscussionManager(this.#lecture);
     this.#currentTimeMs = 0;
     this.#currentRootCommentElements = [];
     this.#nearestComments = [];
   }
 
   /**
-   * Initialize the discussion area by loading the current comments.
+   * Adds event listener allowing seeking discussion area
+   * on event broadcast.
+   */
+  addSeekingListener() {
+    this.#eventController.addEventListener((timestampMs) => {
+      this.seek(timestampMs);
+    }, 'seek');
+  }
+
+  /**
+   * Adds event listener for seeking and initializes the discussion area by
+   * loading the current comments.
    */
   async initialize() {
+    this.addSeekingListener();
+    // This is used as the `onclick` handler of the new comment area submit
+    // button. It must be set after discussion is initialized.
+    window.postNewComment = this.postNewComment.bind(this);
```

Comment:
Do you need to sync this PR, this code looks unrelated to this change.

Reply:
Synced with base branch